subprojects/packagefiles/liburing/meson.build

```diff
@@ -71,7 +71,16 @@ int main(int argc, char **argv)
 '''
 has_cxx = cpp.compiles(code, name: 'C++')
 
-has_ucontext = cc.has_type('ucontext_t', prefix: '#include <ucontext.h>')
+code = '''#include <ucontext.h>
```

Comment:
I think the goal of this change is because the upstream configure now checks for makecontext vs. makecontext_e2k -- is it possible to just check if the header provides the former symbol instead of reverting to a compile test?

Reply:
I think you are right. It seams that checking if `makecontext` is available is sufficient to match upstream's behavior.

I probably have just seen the diff and not the commit message of https://github.com/axboe/liburing/commit/b5f2347b53d4336da355aeafbb81ee748f8bc600.

What do you think about:

```diff
diff --git a/subprojects/packagefiles/liburing/meson.build b/subprojects/packagefiles/liburing/meson.build
index 17544b1..81b954c 100644
--- a/subprojects/packagefiles/liburing/meson.build
+++ b/subprojects/packagefiles/liburing/meson.build
@@ -71,16 +71,8 @@ int main(int argc, char **argv)
 '''
 has_cxx = cpp.compiles(code, name: 'C++')

-code = '''#include <ucontext.h>
-int main(int argc, char **argv)
-{
-  ucontext_t ctx;
-  getcontext(&ctx);
-  makecontext(&ctx, 0, 0);
-  return 0;
-}
-'''
-has_ucontext = cc.compiles(code, name: 'ucontext')
+has_ucontext = cc.has_type('ucontext_t', prefix: '#include <ucontext.h>')
+  and cc.has_function('makecontext', prefix: '#include <ucontext.h>')

 conf_data = configuration_data()
 conf_data.set('CONFIG_HAVE_KERNEL_RWF_T', has__kernel_rwf_t)
```

Reply:
This seems like it should do the right thing, yeah.